src/router/index.ts

```diff
@@ -28,6 +28,12 @@ const routes: Array<RouteRecordRaw> = [
 		props: true,
 		component: () => import(/* webpackChunkName: "user" */ "../views/UserPage/UserPage.vue"),
 	},
+	{
+		path: "/emote-sets/:setID",
+		name: "EmoteSet",
+		props: true,
+		component: () => import("../views/EmoteSetPage/EmoteSetPage.vue"),
```

Comment:
The comment is actually used by webpack please include it.

Reply:
Im lazy to change *all* components to script setup now, I'll do that over time. Ive been swapping them over as I modify them